reference/api/settings.mdx

```diff
@@ -886,6 +889,103 @@ Reset an index's pagination settings to their [default value](#pagination-object
 
 You can use the returned `taskUid` to get more details on [the status of the task](/reference/api/tasks#get-one-task).
 
+## Proximity precision
+
+Calculating the [proximity ranking rule](/learn/core_concepts/relevancy#3-proximity) is a resource-intensive operation. Lowering its precision may significantly improve performance and will have little impact on result relevancy in most use-cases.
+
+`proximityPrecision` accepts one of the following string values:
+
+- `"byWord"`: calculate precise distance between query terms. Higher precision, but may lead to longer indexing time. This is the default setting
+- `"byAttribute"`: detect if multiple query terms are present in the same attribute. Lower precision, but shorter indexing time
+
+### Get proximity precision settings
+
+<RouteHighlighter method="GET" route="/indexes/{index_uid}/settings/proximity-precision"/>
+
+Get the proximity precision settings of an index.
+
+#### Path parameters
+
+| Name              | Type   | Description                                                            |
+| :---------------- | :----- | :--------------------------------------------------------------------- |
+| **`index_uid`** * | String | [`uid`](/learn/core_concepts/indexes#index-uid) of the requested index |
+
+#### Example
+
+<CodeSamples id="get_proximity_precision_settings_1" />
+
+##### Response: `200 OK`
+
+```json
+{
+  "proximityPrecision": "byWord"
```

Comment:
GET /indexes/{index_uid}/settings/proximity-precision does not return a `JSON object` but a `JSON string`

#### Response: `200 OK`

```json
"byWord"|"byAttribute"
```

Reply:
I opened https://github.com/meilisearch/documentation/issues/2693
thank you for the feedback